### ISSUE TITLE
predicting paratopes

### ISSUE THREAD

**Issue by avilella** [alchemab/antiberta#1]
Hi all, congratulations on the Cell Patterns paper. I am keen to give antiberta a try.

On the notebook, where it says "Inference - how to go from sequence to predicted sequence" at the end, does that give a paratope string prediction similar to the Figure 4 sequences with dots in the paper?

![image](https://user-images.githubusercontent.com/158007/169315441-7dc8e3cb-9dcb-48eb-bf67-b45085b5b352.png)

**Comment by jleem-iso** [comment]
Thanks! No, what it does give is actually the full-length VH (or VL) input sequence you give it with dashes in the middle, so it's something like this:

`-------------------------GSKS---------------------N----------------W-S---D-------`

What I've done with the dots there in the figure is just for visualisation purposes... hope that helps? But the code I wrote in the notebook should convert the logits into class labels of 1s and 0s that you can then use to pick out paratope positions.
